stacks/rds-mediajoint.yml

```diff
@@ -0,0 +1,124 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
The `/stacks` directory has kind of stumbled into being reserved for stacks that are deployed though CD. I suggest we move this to a `/db` directory (to match `/dns` and `/cdn`, which also have stacks that are managed outside of CD)